cloudinit/net/__init__.py

```diff
@@ -121,7 +121,9 @@ def master_is_bridge_or_bond(devname):
         return False
     bonding_path = os.path.join(master_path, "bonding")
     bridge_path = os.path.join(master_path, "bridge")
-    return (os.path.exists(bonding_path) or os.path.exists(bridge_path))
+    ovs_path = sys_dev_path(devname, path="upper_ovs-system")
```

Comment:
Let's pull this out into

```
def master_is_openvswitch(devname):
 ...
```

And then in get_interfaces() we can do something like

```
if get_master(name) is not None:
    if not master_is_bridge_or_bond(name) or not master_is_openvswitch(name):
       continue
```

Reply:
Done.